src/CargoSubsystem.cpp

```diff
@@ -27,7 +27,7 @@ void CargoSubsystem::teleop(){
     if(IsIntaking()){
```

Comment:
Right now if you are not currently intaking, then you will never be able to intake, because you are only checking the joystick buttons if you are intaking.